src/Workspaces/Core/Portable/LanguageServices/SyntaxFactsService/ISyntaxFactsService.cs

```diff
@@ -77,7 +80,9 @@ internal interface ISyntaxFactsService : ILanguageService
         SyntaxNode GetObjectCreationType(SyntaxNode node);
 
         bool IsBinaryExpression(SyntaxNode node);
-        void GetPartsOfBinaryExpression(SyntaxNode node, out SyntaxNode left, out SyntaxNode right);
+        void GetPartsOfBinaryExpression(SyntaxNode node, out SyntaxNode left, out SyntaxNode right, out SyntaxToken token);
+        SyntaxToken GetOperatorTokenOfBinaryExpression(SyntaxNode node);
```

Comment:
this can just be an extension method.  no need for c# or VB to impl it, you can just build it off the GetPartsOfBinaryExpression method.